packages/Teams/dotnet/Actions/SendMessageToTeamsChannel.cs

```diff
@@ -118,22 +119,19 @@ public override async Task<DialogTurnResult> BeginDialogAsync(DialogContext dc,
                 teamsChannelId = dc.Context.Activity.TeamsGetChannelId();
             }
 
-            if (!(dc.Context.Adapter is BotFrameworkAdapter))
```

Comment:
This is a fix I am taking for @EricDahlvang that requires 4.14 version of SDK for the new API call.